plugins/modules/source_control/gitlab/gitlab_group.py

```diff
@@ -162,11 +162,16 @@ def createOrUpdateGroup(self, name, parent, options):
         if self.groupObject is None:
             parent_id = self.getGroupId(parent)
 
-            group = self.createGroup({
+            payload = {
                 'name': name,
                 'path': options['path'],
                 'parent_id': parent_id,
-                'visibility': options['visibility']})
+                'description': options['description'],
```

Comment:
@russoz same error.  
I guess line 169 (the description key) must be left out in the payload. Because with line 172/173, you'll add the key to the payload if description exists.

Reply:
oops, my bad, forgot to remove that bugger

Reply:
```suggestion
```